api/awsauth/awsauth/auth_app.py

```diff
@@ -89,15 +89,22 @@ def _create_api_key(email: str) -> str:
     return uuid.uuid4().hex
 
 
-def _get_or_create_api_key(email):
+def _get_or_create_api_key(email: str, is_crs_user: bool):
```

Comment:
how about using keyword-only parameter... `_get_or_create_api_key(email: str, *, is_crs_user: bool):` ?

Reply:
Is that around in python 3.7?

Reply:
I'll keep it in mind going forward!